src/histogram.js

```diff
@@ -259,9 +266,20 @@ angular.module('grafana.directives').directive('grafanaHistogram', function($roo
           }
         };
 
+        var scopedVars = ctrl.panel.scopedVars;
+        var bucketSize = !panel.bucketSize && panel.bucketSize !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.bucketSize.toString(), scopedVars));
+        var minValue = !panel.minValue && panel.minValue !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.minValue.toString(), scopedVars));
+        var maxValue = !panel.maxValue && panel.maxValue !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.maxValue.toString(), scopedVars));
+
+        switch(panel.bucketMode) {
+          case 'count':
+            bucketSize = (maxValue - minValue) / bucketSize;
```

Comment:
Actualy there is should be logic to calculate bucketSize without min/max values.
But as I don't need that - I didn't inplemented it :)
Only requirement is not to broke Grafana This why there is 1 by default for bucketSize.